docs/network-policy.md

```diff
@@ -1,4 +1,50 @@
-# Cluster Network Policy
+# Antrea Network Policies
+
+## Summary
+
+Antrea supports standard K8s NetworkPolicies to secure traffic between Pods. These
+NetworkPolicies are written from an application developers perspective, hence they
+lack the ability to gain a finer-grained control over the security policies that
+a cluster administrator would require. This document describes a few new CRDs
+supported by Antrea to provide the administrator with more control over
+security within the cluster and which are meant to co-exist with and complement the
+K8s NetworkPolicy.
+
+## Tier
+
+Antrea supports grouping Antrea NetworkPolicies together in a tiered fashion
+to provide a hierarchy of security policies. This is achieved by setting the
+Antrea NetworkPolicies, like a ClusterNetworkPolicy, with the name of the
+`tier` to which it is associated. Each tier has a priority associated with it,
+which determines its relative order among other tiers.
+
+**Note**: K8s NetworkPolicies will be evaluated once all tiers have been evaluated.
+
+### Static tiers
+
+Currently, we support 5 static tiers in Antrea. They are as follows in the
+relative order of precedence:
+
+    Emergency > SecurityOps > NetworkOps > Platform > Application  
```

Comment:
I'm a bit afraid that this won't render well with netlify (website), so maybe use a text code snippet (surrounded by empty lines):
```text
Emergency > SecurityOps > NetworkOps > Platform > Application  
```
I haven't checked though, so this is just me being paranoid